components/webfield/ProgramChairConsole.js

```diff
@@ -714,6 +770,18 @@ const ProgramChairConsole = ({ appContext }) => {
                 : areaChair.areaChairProfileId,
             }
           }),
+          secondaryAreaChairs: secondaryAreaChairs.map((areaChair) => {
+            const profile = secondaryAreaChairProfiles.find(
+              (p) => p.id === areaChair.areaChairProfileId
+            )?.profile
+            return {
+              ...areaChair,
+              preferredName: profile ? getProfileName(profile) : areaChair.areaChairProfileId,
```

Comment:
I think if the profile is missing we should use prettyId here:

```suggestion
              preferredName: profile ? getProfileName(profile) : prettyId(areaChair.areaChairProfileId),
```

Reply:
I see in the PC console screenshot above it's displaying the tilde id.